### PR TITLE
Add support for multiple responses upon the same request towards southbound

### PR DESCRIPTION
**Pull request by rafaelcoelho**
- Add support for multiple responses for the same key
in order to replay the correct data in sequential.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,4 +1,6 @@
 {
+  "multipleResponseEnable": true,
```
Comment (rafaelcoelho) [inline config.json:2]: Update the [README.md](https://github.com/rafaelcoelho/jsproxy/blob/master/README.md) about those new configurations.

**Review by rafaelcoelho — COMMENTED** [review]
Update documentation

**Review by lucas-jc — APPROVED** [review]
I'm ok with the implementation with some debits:
 - We need to run the check style in order to organize the code (e.g. many useless semicolons)
 - Improve the documentation to explain the use of the modes recorder and dual
 - Unit tests

**Review by lucas-jc — APPROVED** [review]
All ok!

**Pull request merged**